tox.ini

```diff
@@ -1,40 +1,41 @@
 [tox]
 envlist =
-    cov-clean
-    cov-combine
-    cov-report
+    lint
+    mypy
+    test-lazy-imports
+    coverage_clean
     py{311,310,39,38,37,36}
     py36-mindeps
+    coverage_report
 skip_missing_interpreters = true
-minversion = 3.0.0
+# TODO: set a minversion after we drop support for python3.6
+# minversion = 4.0.0
 
 [testenv]
-usedevelop = true
+# build a wheel, not a tarball, and use a common env to do it (so that the wheel is shared)
+package = wheel
+wheel_build_env = build_wheel
+
 extras = dev
 deps =
     mindeps: requests==2.19.1
     mindeps: pyjwt==2.0.0
     mindeps: cryptography==3.3.1
     mindeps: typing_extensions==4.0
-commands = coverage run -m pytest {posargs}
+commands = coverage run --source globus_sdk -m pytest {posargs}
```

Comment:
Why is `--source globus_sdk` needed? Is the config in `.coveragerc` not producing expected results?

Reply:
🤦 It's not needed. I think I was toying with it earlier and didn't remove it.